index.html

```diff
@@ -2278,9 +2314,9 @@ <h4 id="multibase">The `multibase` Datatype</h3>
             </dd>
             <dt>The lexical space</dt>
             <dd>
-Any string that starts with a [[?MULTIBASE]] character and the rest of
-the characters consist of allowable characters in the respective base-encoding
-alphabet.
+Any string that starts with a <a href="#multibase">Multibase</a> character and
```

Comment:
Please define what the multibase characters are.

Reply:
That is done here:

https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/196.html#multibase-0

If you disagree, please provide concrete text regarding what you'd like to see.